### ISSUE TITLE
Errors from ReorderableList with Fungus 3.7.0

### ISSUE THREAD

**Issue by kankaristo** [snozbot/fungus#649]
I recently updated Fungus to 3.7.0, but I get errors from Rotorz.ReorderableList when I try to view a flowchart (prefab) in the inspector.

I "downgraded" ReorderableList to the one that is shipped with Fungus 3.6.0 and everything's working again.

Was ReorderableList upgraded to a new version with Fungus 3.7.0 (there's nothing in the changelog), or is the release somehow corrupted?

I'm using Unity 5.6.2, which might also be causing the issue.

The error that I'm getting:
```
TypeLoadException: Could not load type 'UnityEngine.ImageConversion' from assembly 'Editor.ReorderableList'.
Rotorz.ReorderableList.Internal.ReorderableListResources..cctor ()
Rethrow as TypeInitializationException: An exception was thrown by the type initializer for Rotorz.ReorderableList.Internal.ReorderableListResources
Rotorz.ReorderableList.ReorderableListStyles..cctor ()
Rethrow as TypeInitializationException: An exception was thrown by the type initializer for Rotorz.ReorderableList.ReorderableListStyles
Rotorz.ReorderableList.ReorderableListControl..ctor ()
Rotorz.ReorderableList.ReorderableListGUI..cctor ()
Rethrow as TypeInitializationException: An exception was thrown by the type initializer for Rotorz.ReorderableList.ReorderableListGUI
Fungus.EditorUtils.FlowchartEditor.OnInspectorGUI () (at Assets/3rdParty/Fungus/Scripts/Editor/FlowchartEditor.cs:76)
UnityEditor.InspectorWindow.DrawEditor (UnityEditor.Editor editor, Int32 editorIndex, Boolean rebuildOptimizedGUIBlock, System.Boolean& showImportedObjectBarNext, UnityEngine.Rect& importedObjectBarRect) (at /home/builduser/buildslave/unity/build/Editor/Mono/Inspector/InspectorWindow.cs:1229)
UnityEditor.DockArea:OnGUI()
```

**Comment by ciarandavies7191** [comment]
From a clean installation into Unity project of v3.7.0, I get build errors under Visual Studio for missing namespace:
"The type or namespace name 'Rotorz' could not be found (are you missing a using directive or an assembly reference?)"
Project is Unity.Editor and installation worked under previous Fungus versions.

**Comment by stevehalliwell** [comment]
There were some changes in Unity 2017.x that necessitated that the Rotorz Reorderable list be updated to a newer version, which is then incompatible with Unity versions below 2017. I believe it is possible to get Fungus 3.7 to work in older versions of Unity but it would require you to manually downgrade the rotorz reorderable list dll that comes with fungus. 
I imagine there are also some scripts that make use of newer features and that were updated due to deprecation.

**Comment by kankaristo** [comment]
I'm using Fungus 3.7.0 and the version of ReorderableList that was shipped with Fungus 3.6.0, and everything seems to be working well with Unity 5.6.x.

I was just a bit confused, because Fungus' change log didn't say anything about the updated ReorderableList. But, this is solved for me.

Hopefully we can update Unity at some point, but 2017.1 has really long load times on Android builds, while 2017.2 broke Unity IAP on Linux, so about half of our team can't open the 2017.2 editor. Fingers crossed for 2017.3. :stuck_out_tongue:

@ciarandavies7191, are you using an old version of Unity?

**Comment by ciarandavies7191** [comment]
@kankaristo I'm using Unity 2017.1.1 which had a working version of Fungus v3.6.0 without issues. It was only after upgrading to 3.7 that I ran into issues with ReorderableList.
I have tried using both Fungus/ReorderableList as source-builds but that didn't work either
